src/components/VideoPlayer/VideoPlayerControls/VolumeButton/index.tsx

```diff
@@ -10,18 +11,13 @@ import {useVolumeContext} from '@components/VideoPlayerContexts/VolumeContext';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as NumberUtils from '@libs/NumberUtils';
-import stylePropTypes from '@styles/stylePropTypes';
 
-const propTypes = {
-    style: stylePropTypes.isRequired,
-    small: PropTypes.bool,
+type VolumeButtonProps = {
+    style: ViewStyle;
```

Comment:
We should use `StyleProp` for style types 😄 

```suggestion
    style: StyleProp<ViewStyle>;
```